examples/brushing/deckgl-overlay.js

```diff
@@ -136,7 +136,9 @@ export default class DeckGLOverlay extends Component {
       opacity, mouseEntered, mousePosition} = this.props;
     const {arcs, targets, sources} = this.state;
 
-    const startBrushing = Boolean(mouseEntered && enableBrushing);
+    // mouseEntered is undefined when mouse is in the component while it first loads
```

Comment:
Not sure about using a "three-state" boolean, seems like an easy thing for a maintainer to miss... 

Consider tracking using another Boolean  variable instead?